### ISSUE TITLE
Huawei B535-232 mod fw

### ISSUE THREAD

**Issue by proudturk** [Huawei-LTE-routers-mods/huawei_balong_modfw_kitchen#17]
Hello.

Is mod fw possible for Huawei B535-232?

Thank you

**Comment by jhinzuo** [comment]
Was wondering the same, i know the the kitchen support V7R2 but does that include all devices? like V7R22 (Hi6931) - B528S-23a.

Would be grateful for a reply.

**Comment by ProxVirtualZone** [comment]
Yeah, it's really can be helpful if I can mod my 535-232 for fixing TTL and changing imei
